lib/nerves_hub_link/certificate.ex

```diff
@@ -5,20 +5,6 @@ defmodule NervesHubLink.Certificate do
   @public_keys Application.get_env(:nerves_hub_link, :fwup_public_keys, [])
                |> NervesHubCLI.resolve_fwup_public_keys(org)
 
-  ca_cert_path =
-    System.get_env("NERVES_HUB_CA_CERTS") || Application.get_env(:nerves_hub_link, :ca_certs) ||
-      Path.join([File.cwd!(), "ssl", "prod"])
-
-  ca_certs =
-    ca_cert_path
-    |> File.ls!()
-    |> Enum.map(&File.read!(Path.join(ca_cert_path, &1)))
-    |> Enum.map(&X509.Certificate.to_der(X509.Certificate.from_pem!(&1)))
-
-  if ca_certs == [], do: raise("Couldn't find NervesHub CA certificates in '#{ca_cert_path}'")
-
-  @ca_certs ca_certs
-
```

Comment:
If we want to be really safe, we should prob check config options at compile time as well:

```elixir
ca_store = Application.get_env(:nerves_hub_link, :ca_store)

if is_function(ca_store) and not is_list(ca_store.()) do
 raise(":ca_store function must return a list of DER encoded certificates")
end

if is_module(ca_store) and not function_exported?(ca_store, :cacerts, 0) do
  raise("#{inspect(ca_store)}.cacerts/0 is undefined
end
```

Reply:
Scratch this. It won't work as a dependency of other things 😢